.github/workflows/swift-toolchain.yml

```diff
@@ -1322,7 +1345,10 @@ jobs:
                 -D USE_WIN32_LARGE_FILES=${{ matrix.os == 'Windows' && 'YES' || 'NO' }} `
                 -D USE_WIN32_LDAP=NO `
                 -D ZLIB_ROOT=${{ github.workspace }}/BuildRoot/Library/zlib-1.3/usr  `
-                -D ZLIB_LIBRARY=${{ github.workspace }}/BuildRoot/Library/zlib-1.3/usr/lib/zlibstatic.lib
+                -D ZLIB_LIBRARY=${{ github.workspace }}/BuildRoot/Library/zlib-1.3/usr/lib/zlibstatic.lib `
+                -D CMAKE_POSITION_INDEPENDENT_CODE=${{ matrix.os == 'Android' && 'YES' || 'NO' }} `
```

Comment:
I think that we can set this unconditionally always, can't we?

Reply:
Done.